pcap/stream.go

```diff
@@ -314,6 +333,32 @@ func (c *tcpStream) Accept(tcp *layers.TCP, _ gopacket.CaptureInfo, dir reassemb
 		}
 	}
 
+	// One of the flows initiated a connection close request
+	if tcp.FIN {
```

Comment:
I do think it is important to handle RST too, or at least leave a FIXME about abnormal termination.

If the server crashes hard (a kernel failure) or there is a load balancer problem directing traffic to the wrong server, then the client will get a RST back instead of a FIN.  Of course, we might not be still around to see the traffic in that case.

The server getting a RST is a bit more obscure.

The comments here do not make it clear how we handle the FIN-ACK from the side that did not initiate the shutdown.